### PR TITLE
Signup/login pages

### PR DESCRIPTION
**Pull request by MilesDev91**
### What this PR does (required):
- Styled the Signup and Login pages to match the mock! Changed the MUI theme which will affect the rest of the application.

### Screenshots / Videos (front-end only):
Screenshot (40)

### Any information needed to test this feature (required):
- Everything should be responsive. Errors still function as normal. Click into the fields and try to submit with the wrong information.

### Any issues with the current functionality (optional):
- The forgot? text is still in blue. Will address (if needed) in the future.
- Wanted to have the forms prefilled (I think it looks nicer that way) but I was just short of figuring it out. Figured it was a better use of my time to continue and revisit in the future if I have time.

### CODE REVIEW

File: client/src/pages/Login/Login.tsx [hunk 1]
```diff
@@ -38,25 +38,18 @@ export default function Login(): JSX.Element {
...
-          className={classes.authWrapper}
-        >
-          <AuthHeader linkTo="/signup" asideText="Don't have an account?" btnText="Create account" />
+        <Box display="flex" alignItems="center" flexDirection="column" className={classes.authWrapper}>
           <Box width="100%" maxWidth={450} p={3} alignSelf="center">
             <Grid container>
               <Grid item xs>
                 <Typography className={classes.welcome} component="h1" variant="h5">
-                  Welcome back!
+                  <strong>Welcome back!</strong>
```
Comment (rajivtitus) [inline client/src/pages/Login/Login.tsx]: Alternatively, you can apply a `font-weight` using the existing CSS class

File: client/src/pages/Login/LoginForm/LoginForm.tsx [hunk 2]
```diff
@@ -48,48 +48,47 @@ export default function Login({ handleSubmit }: Props): JSX.Element {
         <form onSubmit={handleSubmit} className={classes.form} noValidate>
           <TextField
             id="email"
-            label={<Typography className={classes.label}>E-mail address</Typography>}
+            label={<Typography className={classes.label}>EMAIL ADDRESS</Typography>}
```
Comment (rajivtitus) [inline client/src/pages/Login/LoginForm/LoginForm.tsx]: Please avoid using CAPITALCASE text as some screen readers interpret them as abbreviations. You can use a CSS class to transform the text case instead :)

Ditto throughout!

File: client/src/pages/SignUp/SignUp.tsx [hunk 3]
```diff
@@ -39,25 +39,18 @@ export default function Register(): JSX.Element {
...
-          className={classes.authWrapper}
-        >
-          <AuthHeader linkTo="/login" asideText="Already have an account?" btnText="Login" />
+        <Box display="flex" alignItems="center" flexDirection="column" className={classes.authWrapper}>
           <Box width="100%" maxWidth={450} p={3} alignSelf="center">
             <Grid container>
               <Grid item xs>
                 <Typography className={classes.welcome} component="h1" variant="h5">
-                  Create an account
+                  <strong>Sign up</strong>
```
Comment (rajivtitus) [inline client/src/pages/SignUp/SignUp.tsx]: Same as comment above about usage of a CSS class instead

File: client/src/themes/theme.ts [hunk 4]
```diff
@@ -2,18 +2,45 @@ import { createTheme } from '@mui/material/styles';
...
+    fontFamily: '"Roboto", "Arial", "Open Sans", "sans-serif"',
     fontSize: 12,
     button: {
       textTransform: 'none',
       fontWeight: 700,
     },
   },
   palette: {
-    primary: { main: '#3A8DFF' },
+    primary: { main: '#f14140' },
```
Comment (rajivtitus) [inline client/src/themes/theme.ts:13]: 👍🏽

File: client/src/pages/SignUp/SignUp.tsx [hunk 5]
```diff
@@ -39,25 +39,18 @@ export default function Register(): JSX.Element {
...
                 </Typography>
               </Grid>
             </Grid>
             <SignUpForm handleSubmit={handleSubmit} />
           </Box>
-          <Box p={1} alignSelf="center" />
+          <AuthHeader linkTo="/login" asideText="Already a member?" linkText="Login" />
```
Comment (rajivtitus) [inline client/src/pages/SignUp/SignUp.tsx]: Can we rename this component to AuthFooter as we are using it to the bottom of the form?

**Review by rajivtitus — COMMENTED** [review]
Great work with this! Just some minor comments to be address :)

File: client/src/pages/Login/Login.tsx (hunk 1 above)
Comment (MilesDev91) [reply client/src/pages/Login/Login.tsx]: Ya this was before I figured out how to override MUI root classes without affecting the theme. I'll address this and other changes!

**Comment by MilesDev91** [comment]
Fixed the concerns!

**Review by moffatethan — APPROVED** [review]
Good to go @MilesDev91 @rajivtitus 

🚀 💯

**Pull request merged**